src/components/core/Flex/Flex.js

```diff
@@ -1,28 +1,20 @@
 import styled from 'styled-components';
-import {
-	flexWrap,
-	flexDirection,
-	alignItems,
-	justifyContent,
-} from 'styled-system';
 import Box from '../Box';
+import propTypes from '@styled-system/prop-types';
 
-const Flex = styled(Box)(
-	{
-		display: 'flex',
-	},
-	flexWrap,
-	flexDirection,
-	alignItems,
-	justifyContent
-);
+export const Flex = styled(Box)({});
+
+Flex.defaultProps = {
+	display: 'flex',
```

Comment:
I'd put `display: 'flex'` to the definition of the component, not `defaultProps` since it can't be changed.